### ISSUE TITLE
The accuracy of BaseModel is 88.5 when the noise rate is 0.4?

### ISSUE THREAD

**Issue by lzm10214066** [xjtushujun/meta-weight-net#1]
I trained a WRN-28-10 network on cifar-10 with noise rate of 0.4 under uniform noise following the setting in the paper for a total of 40 epochs,  but the accuracy of BaseModel is 88.5, which is really high compared with the results in Table 2.  I don't know what the problem is.

**Comment by xjtushujun** [comment]
The BaseModel is following L2RW's setting：the learning rate decays at 40K and 50K steps, for a total of 60K steps.

**Comment by lzm10214066** [comment]
I think the truth is probably that the network fits the noisy data points only in later learning stages and the noisy data points don't harm the learning process in early stages.

**Comment by wangjksjtu** [comment]
@xjtushujun @lzm10214066, Thanks for your interest work! Yeah, I also trained the BaseModel following your setting (the same as meta-weight-net). Specifically, the learning rate decays at 18K and 19K steps and the total training steps are 20K. The accuracy of baseline model is 88.9 (compared to 70.77 reported in the paper)

BaseModel:
![baseline](https://user-images.githubusercontent.com/22170175/68037966-43436280-fc9f-11e9-896d-ae661b2ea277.png)

Meta-Weight-Net:
![meta_weight_net](https://user-images.githubusercontent.com/22170175/68037967-43436280-fc9f-11e9-9838-5dd0587f9e79.png)

I agree that probably the network overfits to the noise in the later stages. So it seems that the effect of Meta-Weight-Net is alleviating the overfitting by re-weighting the noisy examples? But I am  confused that why not adopt the same setting for BaseModel and Meta-Weight-Net for fair comparisons. Also, it seems BaseModel + Early-Stopping could obtain competitive results in this setting.

**Comment by hongxin001** [comment]
Hi, I also have a question about that. In Figure 7, you reported the accuracy of BaseModel in the figure, and it seems that the accuracy would decrease after 80 epochs. However, I reproduce the experiments and found the accuracy can keep at about 88%. Would you please describe the setting for the base model in Figure 7?

**Comment by zzy99** [comment]
In this setting (actually in most cases), the method in this paper is completely useless (87.5% vs baseline 88%).
